x-pack/plugins/data_enhanced/public/search/sessions_mgmt/components/home.tsx

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiButtonEmpty,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiPageBody,
+  EuiPageContent,
+  EuiSpacer,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { HttpStart, IUiSettingsClient } from 'kibana/public';
+import React from 'react';
+import { UISession } from '../../../../common/search/sessions_mgmt';
+import { SearchSessionsMgmtAPI } from '../lib/api';
+import { AsyncSearchIntroDocumentation } from '../lib/documentation';
+import { TableText } from './';
+import { SearchSessionsMgmtTable } from './table';
+
+interface Props {
+  documentation: AsyncSearchIntroDocumentation;
+  api: SearchSessionsMgmtAPI;
+  http: HttpStart;
+  initialTable: UISession[] | null;
+  uiSettings: IUiSettingsClient;
+}
+
+export function SearchSessionsMgmtHome({ documentation, ...tableProps }: Props) {
```

Comment:
(very, very minor) nit (feel free to completely disregard): Having `Home` in the name might end up being confused with the "home" app. Prefer something like `App` (or I'm free to other suggestions).